Colours_And_Fonts_style_v1.css

```diff
@@ -1,18 +1,20 @@
-/* CSS files add styling rules to your content */
+/* CSS files add styling rules to my content */
+/* Use this section to define styles for your HTML elements */
 
 /* here is the link for importing my google fonts */
-@import url('https://fonts.googleapis.com/css2?family=Noto+Serif+Georgian:wght@100..900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&display=swap');

```

Comment:
I changed the font from Noto serif Georgian to Roboto Condenced as I thought it would be more suitable and better for the end users to read the text when they are reading the things they will do for EOTC.